src/services/TestResultsService.ts

```diff
@@ -418,16 +418,18 @@ export class TestResultsService {
       });
   }
 
-  public isMissingRequiredCertificateNumberOnLec(payload: ITestResultPayload): boolean {
-    let bool = false;
+  public checkMandatoryFieldsLEC(payload: ITestResultPayload): boolean {
```

Comment:
boolean-returning function are normally named "isSomething" or "hasSomething". In this case "isMissingMandatoryFieldsLEC"

Reply:
Yes, makes sense. I was initially planning to return all the missing fields.

Reply:
Requested change completed.